zppy/templates/pcmdi_diags/process_sftlf.py

```diff
@@ -0,0 +1,61 @@
+#!/bin/env python
+##################################################################
+# This script attemts to generate land/sea mask for a given input
+##################################################################
+import datetime
+import os
+import sys
+
+import cdms2 as cdm
```

Comment:
we should try our best to not use `cdms`. I think PMP team is working on migrating to xarray/xcdat as well. maybe there is an updated version?